test/Conversion/TorchToLinalg/embedding_bag.mlir

```diff
@@ -0,0 +1,79 @@
+// RUN: torch-mlir-opt <%s -convert-torch-to-linalg -split-input-file -verify-diagnostics | FileCheck %s
+
+// CHECK-LABEL:  func.func @forward(
```

Comment:
this test doesn't have much value. I would omit it.

Reply:
Per https://github.com/llvm/torch-mlir/blob/main/docs/architecture.md#dos-and-donts-for-unit-vs-end-to-end-testing

Reply:
My thinking was that because we can't test this E2E there could at least be a unit test that verifies whether the iterator types are set properly. Would something like this be better then or should I just drop the test
```
// RUN: torch-mlir-opt <%s -convert-torch-to-linalg -split-input-file -verify-diagnostics | FileCheck %s

// CHECK-LABEL:  func.func @forward(
// CHECK-SAME:         %[[ARG0:.*]]: !torch.vtensor<[?,?],f32>,
// CHECK-SAME:         %[[ARG1:.*]]: !torch.vtensor<[?],si64>,
// CHECK-SAME:         %[[ARG2:.*]]: !torch.vtensor<[?],si64>) -> (!torch.vtensor<[?,?],f32>, !torch.vtensor<[?],si64>, !torch.vtensor<[?],si64>, !torch.vtensor<[?],si64>) {
// CHECK:          %[[BUILTIN_ARG0:.*]] = torch_c.to_builtin_tensor %[[ARG0]] : !torch.vtensor<[?,?],f32> -> tensor<?x?xf32>
// CHECK:          %[[BUILTIN_ARG1:.*]] = torch_c.to_builtin_tensor %[[ARG1]] : !torch.vtensor<[?],si64> -> tensor<?xi64>
// CHECK:          %[[BUILTIN_ARG2:.*]] = torch_c.to_builtin_tensor %[[ARG2]] : !torch.vtensor<[?],si64> -> tensor<?xi64>
// CHECK:          %[[C1:.*]] = arith.constant 1 : index
// CHECK:          %[[DIM1_0:.*]] = tensor.dim %[[BUILTIN_ARG0]], %[[C1]] : tensor<?x?xf32>
// CHECK:          %[[C0:.*]] = arith.constant 0 : index
// CHECK:          %[[DIM0_2_0:.*]] = tensor.dim %[[BUILTIN_ARG2]], %[[C0]] : tensor<?xi64>
// CHECK:          %[[INIT_TENSOR0:.*]] = linalg.init_tensor [%[[DIM0_2_0]], %[[DIM1_0]]] : tensor<?x?xf32>
// CHECK:          %[[CST:.*]] = arith.constant 0.000000e+00 : f32
// CHECK:          %[[ZEROFILL0:.*]] = linalg.fill ins(%[[CST]] : f32) outs(%[[INIT_TENSOR0]] : tensor<?x?xf32>) -> tensor<?x?xf32>
// CHECK:          %[[OUT:.*]] = linalg.generic {indexing_maps = [#map0, #map1, #map2], iterator_types = ["parallel", "reduction", "parallel"]} ins(%[[BUILTIN_ARG1]], %[[BUILTIN_ARG2]] : tensor<?xi64>, tensor<?xi64>) outs(%[[ZEROFILL0]] : tensor<?x?xf32>) {
func.func @forward(%arg0: !torch.vtensor<[?,?],f32>, %arg1: !torch.vtensor<[?],si64>, %arg2: !torch.vtensor<[?],si64>) -> (!torch.vtensor<[?,?],f32>, !torch.vtensor<[?],si64>, !torch.vtensor<[?],si64>, !torch.vtensor<[?],si64>) {

    %int0 = torch.constant.int 0
    %false = torch.constant.bool false
    %none = torch.constant.none
    %result0, %result1, %result2, %result3 = torch.aten.embedding_bag.padding_idx %arg0, %arg1, %arg2, %false, %int0, %false, %none, %false, %none : !torch.vtensor<[?,?],f32>, !torch.vtensor<[?],si64>, !torch.vtensor<[?],si64>, !torch.bool, !torch.int, !torch.bool, !torch.none, !torch.bool, !torch.none -> !torch.vtensor<[?,?],f32>, !torch.vtensor<[?],si64>, !torch.vtensor<[?],si64>, !torch.vtensor<[?],si64>
    return %result0, %result1, %result2, %result3 : !torch.vtensor<[?,?],f32>, !torch.vtensor<[?],si64>, !torch.vtensor<[?],si64>, !torch.vtensor<[?],si64>
  }

```

Reply:
I still don't think there's much value

```
{getParallelIteratorTypeName(),
                                          getReductionIteratorTypeName(),
                                          getParallelIteratorTypeName()}
```

-->

```
CHECK: parallel, reduction, parallel
```

Is just redundant.